targets/f7/ble_glue/gap.c

```diff
@@ -309,6 +315,11 @@ BleEventFlowStatus ble_event_app_notification(void* pckt) {
     return BleEventFlowEnable;
 }
 
+void gap_wait_for_tx_pool_available(FuriWait wait) {
+    furi_check(gap);
+    furi_check(furi_semaphore_acquire(gap->tx_pool_busy, wait) == FuriStatusOk);
```

Comment:
The only place where you acquire semaphore, which means that it will be always full